ausbills/parliament/federal.py

```diff
@@ -336,9 +318,13 @@ def get_bill_em_links(self) -> dict:
         if tr is None:
             return dict()
         links = list(tr.find_all('td')[1].find_all('a'))
-        return {DOC: links[0]['href'],
-                PDF: links[1]['href'],
-                HTML: links[2]['href']}
+        return [
```

Comment:
This function should return a `dict`, but here it's returning a `List[dict]`. It can also return an empty `dict()` a few lines above, so I'm not sure which it's meant to be.